net/net.go

```diff
@@ -305,11 +305,32 @@ func (n *net) GetThread(_ context.Context, id thread.ID, opts ...core.ThreadOpti
 	if _, err = thread.ValidateToken(n.getPrivKey(), args.Token); err != nil {
 		return
 	}
-	return n.store.GetThread(id)
+	return n.getThreadWithAddrs(id)
 }
 
-func (n *net) getThread(_ context.Context, id thread.ID) (thread.Info, error) {
-	return n.store.GetThread(id)
+func (n *net) getThreadWithAddrs(id thread.ID) (info thread.Info, err error) {
```

Comment:
It seemed you'd want to populate `thread.Info` results with address information in a few cases... thread create, add, and get. In those cases above, we just call this helper method that gets the `thread.Info` from the logstore then augments it with address information.